src/SQLite.Net/Interop/ReflectionService.cs

```diff
@@ -33,15 +33,45 @@ public static class ReflectionService
         /// Returns the set of public non-static non-initonly fields followed by the public non-static properties with
         /// public get and set methods.
         /// </summary>
-        public static IEnumerable<MemberInfo> GetPublicInstanceProperties(Type mappedType)
+        public static IEnumerable<MemberInfo> GetPublicInstanceProperties(Type mappedType, IColumnInformationProvider provider)
         {
             var properties = mappedType.GetTypeInfo().GetRuntimeProperties()
                 .Where(p => p.CanRead && p.CanWrite && p.GetMethod.IsPublic && p.SetMethod.IsPublic &&
                             !p.GetMethod.IsStatic && !p.SetMethod.IsStatic);
             var fields = mappedType.GetTypeInfo().GetRuntimeFields()
                 .Where(f => f.IsPublic && !f.IsStatic && !f.IsInitOnly);
 
-            return fields.Union<MemberInfo>(properties).ToList();
+            var members = fields
+                .Union<MemberInfo>(properties).ToList();
+            members.Sort((l, r) =>
```

Comment:
can you comment and summarize the sorting rules/behavior here

Reply:
Resolved in PR #10